packages/frontpage/lib/data/atproto/post.ts

```diff
@@ -57,12 +61,12 @@ export async function getPost({ rkey, repo }: { rkey: string; repo: DID }) {
     throw new DataLayerError("Failed to get service url");
   }
 
-  const { value } = await atprotoGetRecord({
+  const { value, cid } = await atprotoGetRecord({
     serviceEndpoint: service,
     repo,
     collection: PostCollection,
     rkey,
   });
 
-  return PostRecord.parse(value);
+  return { ...PostRecord.parse(value), cid };
```

Comment:
Let's return this in the same form as we get it from the repo
```suggestion
  return { value: PostRecord.parse(value), cid };
```